netket/machine/jax.py

```diff
@@ -324,6 +324,165 @@ def JaxRbm(hilbert, alpha, dtype=complex):
     )
 
 
+def MPSPeriodic(
+    hilbert, bond_dim, diag=False, symperiod=-1, dtype=complex, outdtype=complex
+):
+    r"""
+    Constructs a periodic Matrix Product State (MPS) for a quantum state of discrete
+    degrees of freedom, wrapped as Jax machine.  The MPS is defined as
+
+    .. math:: \Psi(s_1,\dots s_N) = \Tr(A[s_1]\dots A[s_N]),
+
+    for arbitrary local quantum numbers :math:`s_i`, where :math:`A[s_1]` is a matrix
+    of dimension (bdim,bdim), depending on the value of the local quantum number :math:`s_i`.
+
+        Args:
+            hilbert: Hilbert space on which the state is defined. Should be a
+                subclass of `netket.hilbert.Hilbert`.
+            bond_dim (int): Virtual dimension of the MPS tensors.
+            diag (bool): Whether or not to use diagonal matrices in the MPS tensors.
+                default=False
+            symperiod (int): Periodicity in the chain of MPS tensors. For positive
```

Comment:
sorry to be pedantic here, just a (final) small thing: instead of -1, we should use ```symperiod=None``` as default value, it is consistent with all the other classes and less error prone!